apps/src/craft/code-connection/craft.js

```diff
@@ -218,7 +218,11 @@ export const executeUserCode = function (client, code) {
 
   // Register async methods
   codegen.asyncFunctionList = Object.values(asyncMethods);
```

Comment:
Should this line be deleted now that we pass `asyncFunctionList` into `evalWith` as an option?